tests/smtlib/testfile-steps-bound.dolmen.expected

```diff
@@ -0,0 +1,5 @@
+
+unknown
+(:reason-unknown Step limit: 3)
```

Comment:
This is not a valid syntax. This should be `(:reason-unknown s)` where `s` is a valid s-expression, "Step limit: 3" is not a valid s-expression. It should be something similar to `(:reason-unknown (:step-limit 3))`.

Reply:
https://github.com/OCamlPro/alt-ergo/issues/966